### PR TITLE
Two new SO terms (`sequence_variant` and `coding_transcript_variant`)

### LINKED ISSUE #4417: Missing SO terms in `scout.constants.so_terms.py`
Hej, I'm tinkering with VEP 111.0 in our wgs pipeline and ran into the following error when adding a VEP111 re-annotated case into scout. 

```
Traceback (most recent call last):                                                                 
  File "/home/worker/app/scout/adapter/mongo/variant_loader.py", line 719, in load_variants        
    nr_inserted = self._load_variants(                                                             
  File "/home/worker/app/scout/adapter/mongo/variant_loader.py", line 436, in _load_variants       
    parsed_variant = parse_variant(                                                                
  File "/home/worker/app/scout/parse/variant/variant.py", line 175, in parse_variant               
    parsed_transcripts = add_gene_and_transcript_info(parsed_variant, variant, vep_header)         
  File "/home/worker/app/scout/parse/variant/variant.py", line 565, in add_gene_and_transcript_info
    for parsed_transcript in parse_transcripts(raw_transcripts):                                   
  File "/home/worker/app/scout/parse/variant/transcript.py", line 80, in parse_transcripts         
    transcript["region_annotations"] = get_regional_annotation(functional_annotations)             
  File "/home/worker/app/scout/parse/variant/transcript.py", line 292, in get_regional_annotation  
    regional_list.append(SO_TERMS[annotation]["region"])                                           
KeyError: 'coding_transcript_variant'                                                              
```

Seems like the `coding_transcript_variant` term together with `sequence_variant` is missing from:

https://github.com/Clinical-Genomics/scout/blob/1e4a09a674e5ea9518df2ac4313da1677505b3fb/scout/constants/so_terms.py#L2-L45

Can I go ahead and add them based on the current severity ranking in http://www.ensembl.org/info/genome/variation/prediction/predicted_data.html ?

### PR DESCRIPTION
**Pull request by alkc**
closes #4417 

This update adds the two missing SO terms `sequence_variant` and `coding_transcript_variant` listed on http://www.ensembl.org/info/genome/variation/prediction/predicted_data.html

The placement in the existing ranking follows that suggested by VEP (as of version 111.0) in the link above.

Please have a closer look at the `region` vals set for the two new consequences. 

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [ ] tests executed by

### CODE REVIEW

File: scout/constants/so_terms.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# These are the valid SO terms with corresponfing severity rank
+# These are the valid SO terms with corresponding severity rank
```
Comment (dnil) [inline scout/constants/so_terms.py:1]: 💯

**Review by dnil — APPROVED** [review]
Thank you!

**Comment by alkc** [comment]
I pulled up a test build with this branch. The sample that was giving me pain loaded without any issues.

**Comment by alkc** [comment]
Are there any other tests pending or can I merge this branch?

**Comment by dnil** [comment]
Since they were previously not used by VEP as I understand it, this should be safe to merge for VEP at or slightly above v107, and for 110/111 its up to you, but I would think it better to be able to load the cases and work out any kinks in the severity ordering from there? The thing we will want to watch out for is new terms masking items that are lower ranked, but on the severe terms list. Then they could be missed by "clinical" or other pre-saved filters. I don't think this is an issue to you guys, is it?

We will also retest with verification samples when we update VEP on production anyway. As far as I can tell this is good stuff, ready to merge!

**Pull request merged**